lang/de.json

```diff
@@ -74,6 +74,7 @@
     "fire": "Feuer",
     "barrels": "Läufe",
     "jury-rigged": "Gebastelt",
+    "banes": "Banes",
```

Comment:
I have no idea how to translate this without context. Could be "Fluch" as in curse or "Gift" as in toxin. But to me that doesn't make any sense for attacks.

Reply:
The official german translation hasn't even started yet, so maybe we should leave it as is until then.

Reply:
Bane is the opposite to boon. I think in that sense it would be closer to "Fluch". As you said, we can leave it for now! 
(I think it was there as a legacy addition referring to the terminology used in the method. So we could get rid of it from both?)

Reply:
Yeah, then Fluch would be the way to go.
I don't think we have to remove it, if it's official rules terminology. Maybe we'll need it in the future.